src/outputs/alsa.c

```diff
@@ -1014,6 +1020,14 @@ alsa_init(void)
       offset = 44100 * (offset/abs(offset));
     }
 
+  original_adjust = adjust_period_seconds = cfg_getint(cfg_audio, "adjust_period_seconds");
+  if (adjust_period_seconds < 1)
+    adjust_period_seconds = 1;
+  else if (adjust_period_seconds > 20)
+    adjust_period_seconds = 20;
+  if (original_adjust != adjust_period_seconds)
```

Comment:
Maybe better to just return -1 on invalid config

Reply:
I was trying to follow the pattern above this where `offset` is clamped but execution proceeds after a log warning. Is there a reason to have two different approaches?